Composer/packages/client/src/pages/design/index.tsx

```diff
@@ -130,11 +130,12 @@ const DesignPage: React.FC<RouteComponentProps<{ dialogId: string; projectId: st
   }, [dialogId, dialogs, location]);
 
   useEffect(() => {
+    console.log(designPageLocation);
```

Comment:
can you remove this?

Reply:
I shouldn't open PR's before I've had coffee.